src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -7346,6 +7395,17 @@ public void onError(ServerError error)
    }
 
    public TextEditingTargetSpelling getSpellingTarget() { return this.spelling_; }
+   
+   private void nudgeAutosave()
+   {
+      // Cancel any existing autosave timer
+      if (autoSaveTimer_.isRunning())
+         autoSaveTimer_.cancel();
```

Comment:
FWIW you can call `cancel()` directly (it will be a no-op if the timer is not running)

Reply:
That'll save a couple of lines. :-)